### PR TITLE
Localize missing toc message

### PR DESCRIPTION
**Pull request by morpheus-87**
This PR replaces the hardcoded message displayed, when no toc is available, with a localized version.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,17 @@
...
-- npm install
+  - npm install
 # before-script:
-# - grunt test cover
+#   - grunt test cover
 script:
-- npm run travis
+  - npm run travis
+env:
+  - CXX=g++-4.8
```
Comment (aeschylus) [inline .travis.yml]: Why is this necessary to change?

Can we test this change without modifying the Travis build?

File: spec/widgets/toc.test.js [hunk 2]
```diff
@@ -60,7 +60,7 @@ describe('Table of Contents', function() {
       });
 
       expect(this.sandbox.find('.toc')).toExist();
-      expect(this.sandbox.find('h2 span')).toContainText('No index available');
+      expect(this.sandbox.find('h2 span')).toContainText(i18n.t('noIndex'));
```
Comment (aeschylus) [inline spec/widgets/toc.test.js]: Looks like this test is failing because we merged your other PR with the more up-to-date i18next library. Could you make this update, and perhaps shield you local travis.yml?

File: .travis.yml (hunk 1 above)
Comment (morpheus-87) [inline .travis.yml]: These changes come from #1238.

Comment (mejackreed) [inline .travis.yml]: Per https://github.com/ProjectMirador/mirador/pull/1238#issuecomment-281613202 seems like this may no longer be necessary

**Comment by morpheus-87** [comment]
I created a new PR for this issue: #1275, @aeschylus.

File: .travis.yml (hunk 1 above)
Comment (morpheus-87) [inline .travis.yml]: I set up a new PR, that does not contain any changes to the Travis configuration, @aeschylus: #1277.
